docs/migration_guide.rst

```diff
@@ -1,6 +1,23 @@
 Migration guide
 ===============
 
+***************************
+0.28.0 Migration guide
```

Comment:
I wonder if it makes sense to copy all the items from .rc versions as well. But it might make it too cluttered.

Reply:
Yeah let's to that, I think it's a general practice to do so.